staging/src/k8s.io/kube-apiextensions-server/pkg/controller/status/naming_controller.go

```diff
@@ -236,6 +236,31 @@ func (c *NamingConditionController) sync(key string) error {
 		return err
 	}
 
+	crd := &apiextensions.CustomResourceDefinition{}
+	if err := apiextensions.DeepCopy_apiextensions_CustomResourceDefinition(inCustomResourceDefinition, crd, cloner); err != nil {
+		return err
+	}
+
+	for _, condition := range crd.Status.Conditions {
```

Comment:
Not sure I understand the logic behind this loop. I would have expected a `c.requeueAllOtherGroupCRDs(crd)` in line 233 above. Wouldn't that be enough? Of course, we would need some other signature for `requeueAllOtherGroupCRDs` which works with the key (what's the key? namespace/name?)